whitepaper/whitepaper.md

```diff
@@ -706,19 +706,25 @@ Once an API call to Hyperledger Cactus framework is requested by a User, Busines
 Following diagram shows the architecture of Hyperledger Cactus based on the discussion made at Hyperledger Cactus project calls.
 The overall architecture is as the following figure.
 
-<img src="./architecture-with-plugin-and-routing.png" width="700">
+<img src="./cactus_arch.svg" width="700">
 
 ### 5.2.1 Definition of key components in system architecture
 
 Key components are defined as follows:
-- **Application user**: The entity submits API calls to "Cactus Routing Interface". Note: this component is exist outside of Cactus service system.
+- ~~**Application user**: The entity submits API calls to "Cactus Routing Interface". Note: this component is exist outside of Cactus service system.~~
 - **Business Logic Plugin**: The entity executes business logic and provide integration services that are connected with multiple blockchains. The entity is composed by web application or smart contract on a blockchain. The entity is a single plugin and required for executing Hyperledger Cactus applications.
+- **Command API Server (cmd-api-server)**: The server accepts a request from an End-user Application, and return a response depending on the status of the targeted trade. Trade ID will be assigned when a new trade is accepted.
```

Comment:
> The server accepts a request from an End-user Application, and return a response depending on the status of the targeted trade. Trade ID will be assigned when a new trade is accepted.

@sfuji822 Could it be that this description (quoted above) is better suited for a specific plugin than the entirety of the API server? And for the latter, potentially we could phrase it more generically with a focus on it being the host/shell/container/etc. for all the plugins?

Reply:
I did not intend to define generic API servers. I thought this should be named as "CACTUS node server" which provides API interfaces to both end-users and administrators.